src/helpers/__tests__/retry.test.ts

```diff
@@ -0,0 +1,99 @@
+import {AxiosPromise} from 'axios'
+import {retryRequest} from '../retry'
+
+describe('retry', () => {
+  const retryCallback = jest.fn()
+  const createResultWithErrors = (errors: any[]): (() => AxiosPromise) => {
+    let i = -1
+
+    return () => {
+      i = i + 1
+      if (errors[i] === undefined) {
+        return Promise.resolve({
+          config: {},
+          data: {},
+          headers: undefined,
+          status: 200,
+          statusText: '',
+        })
+      } else {
+        return Promise.reject(errors[i])
+      }
```

Comment:
```suggestion
      }
      return Promise.reject(errors[i])
```
no need for else here